cirq/__init__.py

```diff
@@ -168,13 +168,24 @@
     moment_by_moment_schedule,
 )
 
+from cirq.sim import (
+    check_state,
+    decode_initial_state,
+    pretty_state,
```

Comment:
`cirq.pretty_state` is ambiguous. Perhaps `cirq.describe_state` or `cirq.summarize_state_vector`?

Reply:
`dirac_notation` is better than my suggestions.